lib/pure/encodings.nim

```diff
@@ -285,7 +285,7 @@ else:
   when defined(bsd):
     {.pragma: importIconv, cdecl, header: "<iconv.h>".}
     when defined(openbsd):
-      {.passL: "-I/usr/local/include".}
+      {.passL: "/usr/local/include/libiconv.a".}
```

Comment:
is this path correct? why wouldn't it be "/usr/local/lib/libiconv.a ? and should that instead be `passl:-liconv` ?

Reply:
my mistake

Reply:
not sure whether `passl:-liconv` works, it is tricky